playbooks/roles/kibana/defaults/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+# Copyright 2014, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kibana_root: /opt/kibana
+kibana_url: "https://download.elasticsearch.org/kibana/kibana/kibana-3.1.0.tar.gz"
```

Comment:
Maybe we should up it to kibana 4? https://download.elastic.co/kibana/kibana/kibana-4.0.2-linux-x64.tar.gz